.github/workflows/build-artifact.yaml

```
@@ -0,0 +1,66 @@
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-tarball:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the repository
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)